presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -310,6 +310,11 @@ Property Name                                      Description
                                                    can be used to use a different location for each user.
 
 ``hive.translate-hive-views``                      Enable translation for Hive views. (experimental)            ``false``
+
+``hive.legacy-hive-view-translation``              Enable translation for Hive views with CTEs. Set this        ``false``
```

Comment:
This isn't limited to CTEs. I'm not sure we want to document this at all, since this is a deprecated property that can produce incorrect results. This is really a hidden thing for people who were using the old version and understand the consequences.

Reply:
@willmostly confirmed that the documentation is needed.